### PR TITLE
[WWST-6928] Z-Wave Mold Detector DTH

### PR DESCRIPTION
**Pull request by PKacprowiczS**
It's an updated version of the DTH @ZWozniakS has done back in a day https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/38133 
@greens @KKlimczukS

### CODE REVIEW

File: devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy [hunk 1]
```diff
@@ -0,0 +1,205 @@
+/**
+ *  Copyright 2020 SmartThings
```
Comment (greens) [inline devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy:2]: 2021 here and below, right?

File: devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	}
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	// device doesn't send it on inclusion by itslef, so event is needed to populate plugin
+	sendEvent(name: "moldHealthConcern", value: "good", displayed: false)
+
+	def cmds = [
+		secure(zwave.wakeUpV2.wakeUpIntervalSet(seconds: 4 * 3600, nodeid: zwaveHubNodeId)),
```
Comment (greens) [inline devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy]: isn't this sent by default?

**Review by greens — DISMISSED** [review]
otherwise lgtm

File: devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy (hunk 1 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy:2]: Well, Zuza did most of the work back in 2020.

File: devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy (hunk 2 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-mold-detector.src/zwave-mold-detector.groovy]: Yeah, I guess it is.

**Review by greens — APPROVED** [review]

**Pull request merged**